Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -75,33 +75,36 @@ def __init__(self, url: str = '', credentials: dict = None,
         self.cert = (cert_file, key_file) if cert_file and key_file else None
         self.tlp_color = tlp_color
 
-    def build_iterator(self, **kwargs) -> List:
-        results = []
-        for feed_name, feed in self.feed_name_to_config.items():
-            r = requests.get(
-                url=feed.get('url', self.url),
-                verify=self.verify,
-                auth=self.auth,
-                cert=self.cert,
-                headers=self.headers,
-                **kwargs
-            )
-
-            try:
-                r.raise_for_status()
-                data = r.json()
-                result = jmespath.search(expression=feed.get('extractor'), data=data)
-                results.append({feed_name: result})
-
-            except ValueError as VE:
-                raise ValueError(f'Could not parse returned data to Json. \n\nError massage: {VE}')
-
-        return results
-
-
-def test_module(client, params) -> str:
-    client.build_iterator()
-    return 'ok'
+    def build_iterator(self, feed: dict, **kwargs) -> List:
+        r = requests.get(
+            url=feed.get('url', self.url),
+            verify=self.verify,
+            auth=self.auth,
+            cert=self.cert,
+            headers=self.headers,
+            **kwargs
+        )
+
+        try:
+            r.raise_for_status()
+            data = r.json()
+            result = jmespath.search(expression=feed.get('extractor'), data=data)
+
+        except ValueError as VE:
+            raise ValueError(f'Could not parse returned data to Json. \n\nError massage: {VE}')
+
+        return result
+
+
+def test_module(client: Client) -> str:
+    for feed_name, feed in client.feed_name_to_config.items():
+        build_iterator_paging = feed.get('build_iterator_paging')
+        if build_iterator_paging:
+            build_iterator_paging(client, feed)
+        else:
+            client.build_iterator(feed)
+        return 'ok'
```

Comment:
I think you want to return `ok` only once, after all of the iterators for all of the feed config items have been built successfully.